tests/test_bootstrap_wizard.sh

```
@@ -0,0 +1,471 @@
+pass() { echo "PASS: $*"; PASS=$((PASS + 1)); }
+fail() { echo "FAIL: $*" >&2; FAIL=$((FAIL + 1)); }
+
+run_wizard() {
+  # Wrapper for the test environment: skip every preflight check that
+  # depends on real on-disk state, drive prompts via flags, auto-confirm.
+  BOOTSTRAP_SKIP_TOOL_CHECK=1 \
+  BOOTSTRAP_SKIP_MERGEPATH_GUARD=1 \
+  BOOTSTRAP_AUTO_CONFIRM=1 \
+  BOOTSTRAP_AUTO_PROMPT=skip \
+    "$SCRIPT" "$@"
+}
```

Shellcheck (0.11.0)

[info] This function is never invoked. Check usage (or ignored if invoked indirectly).

(SC2329)